test/expo/parser_test.exs

```diff
@@ -0,0 +1,268 @@
+defmodule Expo.ParserTest do
```

Comment:
I added this module to make it nicer to test parsing. I think we should gradually move other tests over here to have a big comprehensive parsing-related test suite.